### DISCUSSION TITLE
V1 sector expiring impact analysis

### DISCUSSION THREAD

**Discussion by Fatman13** [filecoin-project/FIPs#236] [General]
Can we have a V1 sector expiring impact analysis similar to this [one](https://github.com/filecoin-project/FIPs/issues/56#issuecomment-804841505) as network stability is of [high concerns](https://github.com/filecoin-project/FIPs/issues/222#issuecomment-996380314)?

**Comment by kaitlin-beegle** [comment]
Hi @Fatman13! I wanted to give you an update on some work that's happening, related to this. 

Currently, Filecoin Foundation, Protocol Labs' CryptoEconLab, and Block.science are collaborating on a way to better audit FIPs.  Doing so is quite challenge, and necessarily includes finding ways to better understand 1) which FIPs and/or proposals are likely to have the greatest potential impact on the Filecoin network and community, 2) defining what these "impacts" are, and 3) creating a workflow for triaging the most important analyses.  

More to come!
